scripts/app.js

```diff
@@ -71,10 +72,33 @@ function handleLogin(event) {
 
 //add event listener to login
 var loginForm = document.getElementById('login');
+loginForm.addEventListener('submit', handleLogin);
+
+//var x = new Addbeer('corona', '5%', 'lager', 'I mean its beer with lime usually', '2.5');
+//console.log(x);
+//console.log(Addbeer.beerDrink);
+
+var drinkHeader = document.getElementById('table-head');
+var drinkBody = document.getElementById('table-body');
+
+drinkHeader.appendChild(document.createElement('th'));
+
+
+var createHeader = function () {
+  for (var x = 0; x < headerArray.length; x++) {
+    var categories = document.createElement('th');
+    categories.textContent = headerArray[x];
+    drinkHeader.appendChild(categories);
+  }
+};
+
+var createDrinkData = function () {
+  drinkBody.appendChild(userDrinkData);
```

Comment:
in createHeader, add a <tr> to append the <th> tags to. This will give us proper semantic markdown for later when we might need it for extensibility